src/main/java/org/zalando/spearheads/innkeeper/utils/OAuthServiceClient.java

```diff
@@ -0,0 +1,14 @@
+package org.zalando.spearheads.innkeeper.utils;
```

Comment:
it seems that you can't define java annotations in scala... and guice need java annotations :(